config/config.js

```diff
@@ -0,0 +1,11 @@
+require("dotenv").config();
+module.exports = {
```

Comment:
it's generally easier to merge these into on string and have this file look like 

`module.exports = {
  development: {
    use_env_variable: 'SQL_CONNECTION_URL',
  },
  production: {
    use_env_variable: 'SQL_CONNECTION_URL',
  },
}
`

Reply:
Done